docs/root/version_history/current.rst

```diff
@@ -11,6 +11,10 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* http cache filter: http cache filter `getCache` interface changed from returning a reference to
+  returning a shared_ptr - any third-party implementations of this interface will need to be
+  updated accordingly. See changes to `simple_http_cache.cc` and `simple_http_cache.h` in
+  [PR21114](https://github.com/envoyproxy/envoy/pull/21114) for example.
```

Comment:
I don't think this markdown link syntax will render correctly, you need to use sphinx/RST link syntax. See other examples or online docs. Also check formatting CI in general.